core/domain/classifier_services.py

```diff
@@ -424,41 +375,239 @@ def delete_classifier_training_job(job_id):
         classifier_training_job_model.delete()
 
 
-def get_classifier_training_jobs(exp_id, exp_version, state_names):
-    """Gets the classifier training job object from the exploration attributes.
+def get_classifier_training_job(exp_id, exp_version, state_name, algorithm_id):
+    """Gets classifier training job object from exploration attributes.
+
+    Args:
+        exp_id: str. ID of the exploration.
+        exp_version: int. The exploration version.
+        state_name: str. The state names for which we retrieve the job.
+        algorithm_id: int. The ID of the algorithm for which classifier training
```

Comment:
Why int? Isn't it supposed to be a string?

Reply:
Done